deps/chakrashim/chakracore.gyp

```diff
@@ -7,11 +7,10 @@
     'msvs_windows_target_platform_version_prop': '',
     'icu_args%': '',
     'icu_include_path%': '',
+    'icu_lib_path%': '',
     'linker_start_group%': '',
     'linker_end_group%': '',
     'chakra_libs_absolute%': '',
-    'icu_lib_path%': '',
-    'chakra_build_flags': '',
 
     # xplat (non-win32) only
     'chakra_config': 'Release',     # Debug, Release
```

Comment:
@digitalinfinity , I don't feel a need for test build support so far. So not worth while adding it. 
FYI - This was the change I needed to make my `gdb` show locals.